ncm-ipmi/src/main/perl/ipmi.pm

```diff
@@ -29,22 +37,27 @@ sub Configure
       my $passwd = $user->{password};
       my $priv   = $user->{priv};
 
+      #Create and enable user account
       CAF::Process->new([IPMI_EXEC, qw(user set name), $userid, $login],
                         log => $self)->run();
       CAF::Process->new([IPMI_EXEC, qw(user set password), $userid, $passwd],
                         log => $self)->run();
+      CAF::Process->new([IPMI_EXEC, qw(user set priv), $userid, sprintf("0x%X", $priv)],
+                        log => $self)->run();
+      CAF::Process->new([IPMI_EXEC, qw(user enable), $userid],
+                        log => $self)->run();
+      #Set remote access via LAN
+      CAF::Process->new([IPMI_EXEC, qw(channel setaccess), $channel, $userid, qw(link=on ipmi=on callin=on)],
+                        log => $self)->run();
   }
 
-  CAF::Process->new([IPMI_EXEC, qw(mc reset cold)],
-                    log => $self)->run();
+  $self->ConfigureNetwork($channel);
+
+  # Reset BMC to ensure changes have been made active
```

Comment:
how does this ensure it?

Reply:
@Gargravarr2112?

Reply:
i also do a cold reset when making changes, but then i also check if the changes are still present. however, this would make this component very slow.
also if you monitor your bmcs, these will go in error now and then (i don't know how this component works, but it's not stateful i think).

what you could try to do is dump eg `ipmitool lan print` to a semi permanent quattor-related file, and diff it upon a run. if there are actual changes before and after, only then reset and redump/recheck values?

Reply:
but tbh, i don't care that much. i am ok to merge this PR in its current state

Reply:
I like the idea of tracking the state, but I'll open an issue for it.